spec/inquisition/brakeman/runner_spec.rb

```diff
@@ -1,4 +1,38 @@
+require 'ruby_parser/bm_sexp_processor'
+require 'brakeman/processors/lib/safe_call_helper'
+require 'brakeman/warning'
+require 'brakeman/checks/base_check'
+
 RSpec.describe Inquisition::Brakeman::Runner do
+  let(:runner) { described_class.new }
+  let(:high_confidence_warning) { Brakeman::Warning.new(confidence: 0) }
+  let(:medium_confidence_warning) { Brakeman::Warning.new(confidence: 1) }
+  let(:low_confidence_warning) { Brakeman::Warning.new(confidence: 2) }
+  let(:warnings_array) { [high_confidence_warning, medium_confidence_warning, low_confidence_warning] }
+
   describe '#call' do
+    before do
+      allow(Brakeman).to receive_message_chain(:run, :run_checks, :warnings).and_return(warnings_array)
+      allow(high_confidence_warning.file).to receive(:relative)
+    end
+
+    it 'returns instance of array' do
```

Comment:
Maybe instead of 4 specs we can write 1 which will cover all our cases (match_array), what do you think?